src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -451,19 +451,21 @@ function updateDOMProperties(
 
 var ReactDOMFiberComponent = {
 
+  // TODO: Use this to keep track of changes to the host context and use this
+  // to determine whether we switch to svg and back.
+  isNewHostContainer: function(tag : string) {
```

Comment:
Currently there is no way to render a namespace where svg and foreignObject is valid tags (other than HTML and SVG respectively). I'll add a comment about it.